sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -136,6 +136,25 @@ const client = new CertificateClient(url, credential);
 - A **Certificate backup** can be generated from any created certificate. These backups come as
   binary data, and can only be used to regenerate a previously deleted certificate.
 
+## Specifying the Azure Key Vault service API version
+
+By default, this package uses the latest Azure Key Vault service version which is `7.1-preview`. The only other version that is supported is `7.0`. You can change the service version being used by setting the option `apiVersion` in the client constructor as shown below:
```

Comment:
We don't have any plans for releasing anything specific for 7.1-preview, but, yes, we should probably support it since customers will see/know it's available. I'll open separate issues across languages to at least add support.

Reply:
That said, don't target a preview version in GA.

Reply:
> That said, don't target a preview version in GA.

For sure!